docs/modules/hdfs/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -1,6 +1,36 @@
 = Graceful shutdown
 
-Graceful shutdown of HDFS nodes is either not supported by the product itself
-or we have not implemented it yet.
+You can configure the graceful shutdown as described in xref:concepts:operations/graceful_shutdown.adoc[].
 
-Outstanding implementation work for the graceful shutdowns of all products where this functionality is relevant is tracked in https://github.com/stackabletech/issues/issues/357
+== JournalNodes
+
+As a default, JournalNodes have `15 minutes` to shut down gracefully.
+
+The JournalNode process will always run as PID `1` and will receive a `SIGTERM` signal when Kubernetes wants to terminate the Pod.
```

Comment:
Is it really relevant  that the process runs with PID 1 ? This is not true on OpenShift for example.

Reply:
I think our stuff will always run as pid 1 as we don't use `shareProcessNamespace: true`.
I only have your "razvan" OS cluster at hand, and picked a random Pod, OS seems to behave normally
```
kubectl -n openshift-monitoring exec -it alertmanager-main-0 -- bash -c "ls /proc && cat /proc/1/stat"
1   acpi       cgroups   cpuinfo  diskstats  execdomains  fs          ioports   kcore      kmsg         kpageflags  mdstat   modules  net           sched_debug  self      stat   sysrq-trigger  timer_list  version      zoneinfo
60  buddyinfo  cmdline   crypto   dma        fb           interrupts  irq       key-users  kpagecgroup  loadavg     meminfo  mounts   pagetypeinfo  schedstat    slabinfo  swaps  sysvipc        tty         vmallocinfo
66  bus        consoles  devices  driver     filesystems  iomem       kallsyms  keys       kpagecount   locks       misc     mtrr     partitions    scsi         softirqs  sys    thread-self    uptime      vmstat
1 (alertmanager) S 0 1 1 0 -1 1077952768 3591272 0 8 0 136784 67329 0 0 20 0 11 0 5023 1526284288 14384 18446744073709551615 4194304 27613560 140722030886976 0 0 0 1002060288 0 2143420159 0 0 0 17 2 0 0 1 0 0 29711832 30179416 52162560 140722030890179 140722030890741 140722030890741 140722030895078 0
```
We can see the main process is running as pid 1

Reply:
My bad. You are right. I was thinking of user id not process id.

Reply:
Puh, had me sweating for a moment ^^